test/minikube/001-kube.bats

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bats
+#
+# Tests of podman kube commands with minikube
+#
+
+load ../system/helpers.bash
+
+function setup(){
+    # only set up the minikube cluster before the first test
+    if [ "$BATS_TEST_NUMBER" -eq 1 ]; then
+        minikube start
+        echo $output >&2
+    fi
+    # wait for a bit so all the services in the cluster come up
+    sleep 10
+}
+
+###############################################################################
+# BEGIN tests
+
+@test "minikube - check cluster is up" {
+    run minikube kubectl get nodes
+    [ "$status" -eq 0 ]
+    [[ "$output" == *"Ready"* ]]
```

Comment:
Maybe it's my ignorance, but testing `*"Ready"*` seems odd.  Did you mean: `[[ "$output" =~ Ready ]]` (match the regular expression `Ready`)?

If not, checking equality `==` here may be a bit unreliable.  It's conceivable a future version of minikube outputs something slightly different, or a warning message or something causing a flake or annoying-failure.

Reply:
Fixed